core/domain/stats_jobs_one_off_test.py

```diff
@@ -896,3 +896,54 @@ def test_standard_operation(self):
             stats_models.ExplorationStatsModel.get_entity_id(self.EXP_ID, 2))
         model = stats_models.ExplorationStatsModel.get(model_id)
         self.assertEqual(model.num_completions_v2, 3)
+
+
+class TestRegenerateMissingStatsModels(OneOffJobTestBase):
```

Comment:
Just saying -- Please make sure to have 100% test coverage for whatever code you write. In this case I can see RegenerateMissingStatsModels has a few branching statements so please make sure to test all the statements. Doing this is really important. Thanks!

Reply:
Sure @lilithxxx. As I mentioned in the description, I sent the PR out early as I wanted feedback on the job itself. I'll add more assertions, and tests before it is merged!

Reply:
Added more tests for the various cases.